src/TodoListItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+let TodoListItem = (props) => {
+    return (
+        <li>
+            {/* <span>{props.todo.id}</span> */}
+            <span>{props.todo.title}</span>
+        </li>
+    )
+}
```

Comment:
There should be a Remove button here that calls an onRemoveTodo function which should be defined in App.js. Double check you saw everything in the lesson instructions.